### ISSUE TITLE
communication/azure-communication-chat - cspell found spelling errors in source code

### ISSUE THREAD

**Issue by danieljurek** [Azure/azure-sdk-for-python#22637]
Spell check scanning of package at `sdk/communication/azure-communication-chat` detected spelling errors in the source code and related files. This directory is opted out of PR spell checking in PR #22199 to keep PRs unblocked.

## What to do

1. Ensure Node.js is installed (https://nodejs.org/en/download/).
1. Delete the entry in `.vscode/cspell.json`'s `ignorePaths` field. It will look like: `sdk/communication/azure-communication-chat/**`. You need to do this to enable checking the files.
1. From the root of the repo run spell check using `./eng/common/spelling/Invoke-Cspell.ps1 -ScanGlobs "sdk/communication/azure-communication-chat/**"`
1. Fix detections according to http://aka.ms/azsdk/engsys/spellcheck use the "False positives" section to fix false positives
1. Check in changes (including the change to `.vscode/cspell.json` where the `ignorePaths` is updated to remove the entry for this service.). You may need to run `git add -f .vscode/cspell.json` to force adding the changes to the file in git.

## Spell checking output

```
./sdk/communication/azure-communication-chat/azure/communication/chat/_chat_thread_client.py:242:24 - Unknown word (objs)
./sdk/communication/azure-communication-chat/azure/communication/chat/_chat_thread_client.py:242:82 - Unknown word (objs)
./sdk/communication/azure-communication-chat/azure/communication/chat/_chat_thread_client.py:400:24 - Unknown word (objs)
./sdk/communication/azure-communication-chat/azure/communication/chat/_chat_thread_client.py:400:71 - Unknown word (objs)
./sdk/communication/azure-communication-chat/azure/communication/chat/_shared/models.py:28:18 - Unknown word (enviornment)
./sdk/communication/azure-communication-chat/azure/communication/chat/_shared/models.py:32:5 - Unknown word (GCCH)
./sdk/communication/azure-communication-chat/azure/communication/chat/_shared/models.py:32:13 - Unknown word (GCCH)
./sdk/communication/azure-communication-chat/azure/communication/chat/_shared/policy.py:11:46 - Unknown word (IOHTTP)
./sdk/communication/azure-communication-chat/azure/communication/chat/_shared/policy.py:14:33 - Unknown word (IOHTTP)
./sdk/communication/azure-communication-chat/azure/communication/chat/_shared/user_credential_async.py:53:43 - Unknown word (inprogress)
./sdk/communication/azure-communication-chat/azure/communication/chat/_shared/user_credential_async.py:77:26 - Unknown word (inprogress)
./sdk/communication/azure-communication-chat/azure/communication/chat/_shared/user_credential.py:55:37 - Unknown word (inprogress)
./sdk/communication/azure-communication-chat/azure/communication/chat/_shared/user_credential.py:77:20 - Unknown word (inprogress)
./sdk/communication/azure-communication-chat/azure/communication/chat/_shared/utils.py:27:25 - Unknown word (timegm)
./sdk/communication/azure-communication-chat/azure/communication/chat/aio/_chat_thread_client_async.py:236:24 - Unknown word (objs)
./sdk/communication/azure-communication-chat/azure/communication/chat/aio/_chat_thread_client_async.py:236:82 - Unknown word (objs)
./sdk/communication/azure-communication-chat/azure/communication/chat/aio/_chat_thread_client_async.py:392:24 - Unknown word (objs)
./sdk/communication/azure-communication-chat/azure/communication/chat/aio/_chat_thread_client_async.py:392:71 - Unknown word (objs)
./sdk/communication/azure-communication-chat/CHANGELOG.md:58:36 - Unknown word (ontent)
./sdk/communication/azure-communication-chat/tests/_shared/test_communication_identifier_serializer.py:81:25 - Unknown word (someid)
./sdk/communication/azure-communication-chat/tests/_shared/test_communication_identifier_serializer.py:86:96 - Unknown word (someid)
./sdk/communication/azure-communication-chat/tests/_shared/test_communication_identifier_serializer.py:97:25 - Unknown word (someid)
./sdk/communication/azure-communication-chat/tests/_shared/test_communication_identifier_serializer.py:103:58 - Unknown word (someid)
./sdk/communication/azure-communication-chat/tests/_shared/test_utils.py:5:8 - Unknown word (dateutil)
./sdk/communication/azure-communication-chat/tests/_shared/test_utils.py:19:58 - Unknown word (XVCJ)
./sdk/communication/azure-communication-chat/tests/_shared/test_utils.py:20:48 - Unknown word (PBOF)
./sdk/communication/azure-communication-chat/tests/_shared/test_utils.py:25:96 - Unknown word (dateutil)
./sdk/communication/azure-communication-chat/tests/_shared/test_utils.py:25:108 - Unknown word (tzutc)
./sdk/communication/azure-communication-chat/tests/_shared/test_utils.py:31:96 - Unknown word (dateutil)
./sdk/communication/azure-communication-chat/tests/_shared/test_utils.py:31:108 - Unknown word (gettz)
./sdk/communication/azure-communication-chat/tests/_shared/test_utils.py:34:96 - Unknown word (dateutil)
./sdk/communication/azure-communication-chat/tests/_shared/test_utils.py:34:108 - Unknown word (gettz)
./sdk/communication/azure-communication-chat/tests/_shared/test_utils.py:37:108 - Unknown word (gettz)
./sdk/communication/azure-communication-chat/tests/test_chat_client_async.py:30:25 - Unknown word (timegm)
./sdk/communication/azure-communication-chat/tests/test_chat_client_e2e.py:106:18 - Unknown word (thead)
./sdk/communication/azure-communication-chat/tests/test_chat_client_e2e.py:107:48 - Unknown word (thead)
./sdk/communication/azure-communication-chat/tests/test_chat_client.py:31:25 - Unknown word (timegm)
./sdk/communication/azure-communication-chat/tests/test_chat_thread_client_async.py:30:25 - Unknown word (timegm)
./sdk/communication/azure-communication-chat/tests/test_chat_thread_client_e2e_async.py:275:28 - Unknown word (occured)
./sdk/communication/azure-communication-chat/tests/test_chat_thread_client_e2e.py:224:20 - Unknown word (occured)
./sdk/communication/azure-communication-chat/tests/test_chat_thread_client.py:30:25 - Unknown word (timegm)
```

**Comment by petrsvihlik** [comment]
Note: the files in the `_shared` folder are owned by the ACS Auth team and will be fixed as part of #22636 for all modalities at once

**Comment by petrsvihlik** [comment]
Once fixed, we should remove the package from the spellchecker whitelist - #22199
